apps/projects/app/components/Card/Project.js

```diff
@@ -157,8 +153,11 @@ const CardTitle = styled(Text.Block).attrs({
   margin-bottom: 5px;
   text-align: center;
   display: block;
+  /* stylelint-disable-next-line */
   display: -webkit-box;
+  /* stylelint-disable-next-line */
   -webkit-line-clamp: 2;
+  /* stylelint-disable-next-line */
```

Comment:
We already have this sort of `disable-next-line` thing all over the place. Rather than using `--no-verify`, which it seems we'd been doing a bunch for this file, I decided to add these in and make it consistent with the other spots where we use rules like this.

I agree it would be nice to loosen this rule globally and stop making exceptions everywhere.